stable/datadog/templates/daemonset.yaml

```diff
@@ -97,5 +97,7 @@ spec:
       tolerations:
 {{ toYaml .Values.daemonset.tolerations | indent 8 }}
       {{- end }}
+  updateStrategy:
+    type: {{ default "OnDelete" .Values.daemonset.updateStrategy | quote }}
```

Comment:
Nope not missing anything. I am though. PR has been updated with a commented out line to document rolling updates.